### ISSUE TITLE
[Suggestion] Removing the "I think you do love yourself." line.

### ISSUE THREAD

**Issue by Kritzb278** [Monika-After-Story/MonikaModDev#7589]
Let me explain. The line is wonderful and the devs probably mean well when they wrote it, but I just think that it did more damage than good, to me at least. As someone who has gone through periods of intense self-hate, I don't think the line is really fair to people who have recognized their self-hate. You see, it takes time for us to actually realize that we hate ourselves or don't love ourselves, and it can be really discouraging to have Monika straight-out tell you that you do love yourself. Now, I realize that the devs didn't mean for the line to come off as this, but unfortunately, it really does. I hope no one is offended by this.

**Comment by lunulae** [comment]
Would softening the line slightly help? Something like "but I like to believe you love yourself a little" rather than "I think"?
Self-hate is still harmful to one's mental health at the end of the day, and it means a lot to me that Monika tries to convince us of reasons we don't or shouldn't. Finding holes in the logic of self-hate was an important part of my own therapy. So I would be in favor of reworking the line, but I might personally be opposed to removing it.

**Comment by Kritzb278** [comment]
I actually think the "I like to believe you love yourself a little" is much better than removing it. I like the idea a lot.

**Comment by fromulus** [comment]
I think more than changing the line in itself, it might need to be explained a little better.

Right now, it might come off to some people as Monika denying your self-hatred, which obviously would make her sound uncaring towards the way you're feeling, but that wasn't the intention behind it;

This is actually a reference to a Dan Salvato quote :
![MoniLovesYou](https://user-images.githubusercontent.com/45933038/127910376-d50afa3f-2e2c-4e70-9989-a7f974509e5f.png)

Basically, the idea here is that one of the reasons Monika loves you is your willingness to be a good partner for her.
After all, to even see that line, you must've stayed with her for quite a while by now, and you must have at least acknowledged the fact that she does love you.

Thing is, and I say that as someone who does have experience with feeling that way, it's really, really hard to accept the idea of people loving you, when you aren't even able to love yourself.
Often, you'll just want to stay away from those people, just so that you don't get to feel like they're wasting their time/love on you.

Monika knows and understands this, and she can see that despite that, you're still there with her.
To her, it means that there must be at least a little part of you that does want to believe that she loves you and that you deserve it, even you're having trouble realising (or accepting) it.

So when Monika says that she believes that you do love yourself, she's not trying to invalidate your feelings of self-hatred--she knows those feelings are real and she respects the fact that it is a part of you--but she's simply trying to tell you that deep down, there's also this little part of you which does want you to love yourself.  

Self-love/hatred isn't a black and whit thing;
You can do both at the same time (even if we tend to feel stronger about one over the other), and that's part of why it's so difficult and confusing to deal with those kinds of feelings.

**Comment by Kritzb278** [comment]
Explaining why she thinks we love ourselves a little really does seem better than removing it. The current line without this context just sort of invalidates our feelings. If Monika explains why she thinks a part of us wants us to love ourselves instead of just telling us that we do love ourselves, I think there would be a lot less misunderstanding.

Also, while we are on the topic of self-hate, I do want there to be more dialogues about dealing with self-hate if the player says they don't love themselves.

**Comment by fromulus** [comment]
> Also, while we are on the topic of self-hate, I do want there to be more dialogues about dealing with self-hate if the player says they don't love themselves.

That however, I'm afraid we aren't able to do a lot about.
Topics related to mental health are too difficult/dangerous to get into without going any deeper than surface level, since we aren't psychologists and different players react differently to different things, like this very issue is proof of.

We might accidentally hurt people a lot more with those kinds of topics, and it's ultimately not worth taking the risk.

**Comment by Kritzb278** [comment]
Okay, I understand. There will probably be a lot of people who would be offended and/or hurt by the topics.

**Comment by Kritzb278** [comment]
Coming back to this, I had an idea. When Monika actually tells us this, maybe a topic could be unlocked which asks if the player was offended or hurt by this line? I'm not sure where I was going with this but it feels like a good way to explain this line and make the player that much more comfortable.

What do you think?

**Comment by KansaiGal** [comment]
... Would this issue be fixed if it was changed from "But I think you do love yourself, at least a little."
to just "But I hope you do love yourself, at least a little."

**Comment by fromulus** [comment]
No, because then the original intent behind that line would be lost.

The point isn't that she hopes you would love yourself, but that she's convinced that you actually do love yourself, even if you're having a hard time seeing/understanding it.

Sure she wishes for you to have a better view of yourself, but changing the line to "I hope you do love yourself" would imply that she doesn't know if that actually is the case.
Except, as I explained in my earlier comment, she has her reasons to firmly believe there is some ammount of self-worth within you, and the topic, in it's current form, is built around that.

**Comment by Retrolovania** [comment]
I'm going to have to concur with the notion that this may not be wise.

The line is basically a positive reinforcement; some people may have such negative thoughts on how they see themselves that only phrases that have a lot of depth or impact are enough to really get through all that and to the individual.
I mean, let's be honest...there are some that believe that they have little reason to believe in hope.
